packages/contracts/src/dollar/core/CreditNFT.sol

```diff
@@ -1,14 +1,15 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.3;
+pragma solidity 0.8.16;
 
-import "solidity-linked-list/contracts/StructuredLinkedList.sol";
 import "../ERC1155Ubiquity.sol";
+import "solidity-linked-list/contracts/StructuredLinkedList.sol";
 import "./UbiquityDollarManager.sol";
+import "../interfaces/ICreditNFT.sol";
 
-/// @title A Credit NFT redeemable for dollars with an expiry block number
+/// @title A creditNFT redeemable for dollars with an expiry block number
```

Comment:
```suggestion
/// @title A Credit NFT redeemable for Dollars with an expiry block number
```

Perhaps it should be reverted to the old comment style.

Reply:
Changed to `CreditNFT` as that is the contract name. `creditNFT` was inappropriate as that would be a specific instance referenced as a variable.